examples/weather-demo/Cargo.toml

```
@@ -0,0 +1,68 @@
+[package]
+name = "rusty-weather"
+version = "0.1.0"
+authors = ["FELGO GmbH <contact@felgo.com>"]
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+async-std = "1.12.0"
+chrono = "0.4.38"
+directories = "5.0.1"
+log = "0.4.21"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.115"
+
+[target.'cfg(all(not(target_os = "windows"), not(target_os = "android")))'.dependencies]
+slint = { path = "../../api/rs/slint" }
+
+# Using different Slint configuration on Windows to disable the accessibility feature (enabled by default).
+# The feature caused performance issues on Windows (see https://github.com/slint-ui/slint/issues/5338).
+# Note: might be fixed with v1.7
+[target.'cfg(target_os = "windows")'.dependencies]
+slint = { path = "../../api/rs/slint", default-features = false, features = [ "compat-1-2", "backend-winit", "renderer-femtovg" ] }
+
+[target.'cfg(all(not(target_arch = "wasm32"), not(target_os = "android")))'.dependencies]
+env_logger = "0.11.3"
+
+[target.'cfg(target_os = "android")'.dependencies]
+android_logger = "0.14.1"
+openssl = { version = "0.10", features = ["vendored"] }
+slint = { path = "../../api/rs/slint", features = [ "backend-android-activity-06" ] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+openweather_sdk = "0.1.8"
+tokio = "1.37.0"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+console_log = "1.0"
+console_error_panic_hook = "0.1.7"
+wasm-bindgen = "0.2"
+wasm-bindgen-futures = "0.4"
+
+[build-dependencies]
+slint-build = "1.6"
```

You'll need this to compile:

```suggestion
slint-build = { path = "../../api/rs/build" }
```

---

Right, I missed this one! Thanks :slightly_smiling_face: